### PR TITLE
suffix support

### PR DESCRIPTION
**Pull request by arturparkhisenko**
https://github.com/PolymerLabs/polybuild/issues/5

### CODE REVIEW

**Comment by addyosmani** [comment]
This LGTM with the change made. Will wait to hear if @azakus is down with this update.

**Comment by dfreedm** [comment]
LGTM 2, if you can update the README with the new options :)

**Comment by arturparkhisenko** [comment]
@azakus updated ;) tell me if i miss something! And thanks for that awesome Polymer tool (and many other)!

File: README.md [hunk 1]
```diff
@@ -62,11 +62,11 @@ To use "maximum crush" on the command line add the flag `--maximum-crush`
 polybuild --maximum-crush index.html
```
Comment (ebidel) [inline README.md]: How about an example here as well? Along with the output filename that it would produce.

Comment (arturparkhisenko) [inline README.md]: @ebidel , @addyosmani yep thanks, i'll add it tomorrow)

**Comment by ikeagold** [comment]
@azakus already updated, but i can't test it now :bug:, pls if you can

File: README.md (hunk 1 above)
Comment (ebidel) [inline README.md]: Can you add "which will output the file `index.customsuffix.html`" after the cmd line example.

Comment (arturparkhisenko) [inline README.md]: @ebidel thanks, added :white_check_mark:

**Comment by arturparkhisenko** [comment]
@azakus last commit with updated bin file for suffix and updated deps, i tested it, working excellent :yum:, updated CLA too (i'm changed github nickname to realname).

**Comment by addyosmani** [comment]
LGTM for a merge. Thank you, @arturparkhisenko! :cake:

**Pull request merged**